pkg/engine/query/source.go

```diff
@@ -36,7 +36,7 @@ func GetPathToLibrary(platform, relativeBasePath string) string {
 
 	if strings.Contains(platform, "ansible") {
 		return filepath.FromSlash(libraryPath + "/ansible/" + LibraryFileName)
-	} else if strings.Contains(platform, "cloudformation") {
+	} else if strings.Contains(platform, "cloudFormation") {
```

Comment:
should we do an ignore case check in those decision structures?

Reply:
resolved